lisa/analysis/load_tracking.py

```diff
@@ -61,7 +71,7 @@ def _columns_to_drop(cls, event):
         """
         The extra columns not shared between trace event versions
         """
-        if event in ['sched_load_cfs_rq', 'sched_load_se']:
+        if event in ['sched_load_cfs_rq', 'sched_load_se', 'sched_pelt_se']:
```

Comment:
Can't you use the new _SCHED_PELT_SE_NAMES instead here?

Reply:
No, `sched_load_cfs_rq` is part of `_SCHED_PELT_CFS_NAMES`, so it seems to really depend on the specific event. If you have examples of them, we can sort out what fields they actually have.